net.mediaarea.MediaConch.json

```
@@ -68,7 +71,10 @@
       "name": "mediaconch-gui",
       "subdir": "Project/Qt",
       "buildsystem": "qmake",
-      "config-opts": [ "-after", "target.path=/app/bin" ],
+      "build-options": {
+        "env": "QMAKEPATH=/app/lib"
```

this actually is incorrect and flatpak-builder is complaining about it.

```suggestion
        "env": {
            "QMAKEPATH": "/app/lib"
        }
```